### ISSUE TITLE
[NEW CHAPTER] - The Importance of Usability in Accessibility Considerations: Lefthandedness vs Neutrality

### ISSUE THREAD

**Issue by Preshh0** [the-turing-way/the-turing-way#3930]
### Title

The Importance of Usability in Accessibility Considerations: Lefthandedness vs Neutrality

### Guide

Accessibility

### Draft

https://docs.google.com/document/d/1il1krHf_FOOYNiaI5tIVwSaThn3de2Hu_7zKjkUxIpo/edit?usp=sharing

### Summary of proposed chapter

The purpose of the chapter is to raise awareness and generally share knowledge on the effects of designing for mostly right-handed users by default, and how that affects left-handed users. I share some examples of popular tools that mostly favour right-handed users, and some tools that are created to accommodate left-handed users either as specialised tools, or neutral hand tools. 

As a right-hand user, I haven't ever thought of how the products I use affect others who are left-handed. I think as creatives and generally folks who build in science or any other discipline, it is important to understand how different designs affect everyone around us, and in cases where we might not be directly responsible for an outcome, we can talk about it, so those who are responsible, have the knowledge.

### Resources

- https://www.quora.com/How-do-left-handed-people-adapt-to-right-handed-tools

- Gutwinski, S., Lüscher, A., Mahler, L., Kalbitzer, J., Heinz, A., & Bermpohl, F. (2011). Understanding left-handedness. *Dtsch Arztebl International, 108*(50), 849–853. https://doi.org/10.3238/arztebl.2011.0849

- Digital.gov. (2024, March 13). Usability. Digital.gov. https://digital.gov/topics/usability/

- de Kovel, C. G. F., Carrión-Castillo, A., & Francks, C. (2019). A large-scale population study of early life factors influencing left-handedness. *Scientific Reports, 9*(1), 584. https://doi.org/10.1038/s41598-018-37423-8

- Norman, D. A. (2002). The design of everyday things. Basic Books.

### Who can help?

Anyone interested in the topic, and willing to contribute in anyway!

### Updates

- [X] Write chapter outline
- [X] Add material to the chapter
- [X] Combine materials into a readable chapter
- [ ] Proofread
- [ ] Create PR and link in the comments
- [ ] Request reviews
- [ ] Address reviews
- [ ] Merge to main branch

**Comment by aleesteele** [comment]
@Preshh0 - we were discussing this during the Accessibility Working Group meeting today. Would you be able to join the Collaboration Cafe on the 20th of November to discuss with @LizHareDogs and others?

I'll add some notes to the google doc in the meantime - which can be cross-referenced with the pad. 😄

**Comment by Preshh0** [comment]
Yes, absolutely! @aleesteele

**Comment by aleesteele** [comment]
@Preshh0 - just wanted to check-in about this. Can we use the upcoming Accessibility Working Group meetings and/or collaboration cafes to work on this chapter with @LizHareDogs?

**Comment by Preshh0** [comment]
Yes, please. I could not join the previous ones because it conflicted with something else. 

I'd begin working on it ahead of the next call. Thank you, Liz and Anne! ❤️
